app/experimenter/reporting/constants.py

```diff
@@ -0,0 +1,63 @@
+from django.db import models
+
+from experimenter.experiments.constants.legacy import ExperimentConstants
+from experimenter.experiments.constants.nimbus import NimbusConstants
+
+
+class ReportLogConstants(object):
+    EXPERIMENT_STATUS_CHOICES = tuple(
+        sorted(
+            list(
+                set(
+                    tuple(NimbusConstants.Status.choices)
+                    + ExperimentConstants.STATUS_CHOICES
+                )
+            )
+        )
+    )
+    EXPERIMENTS_TYPE = (
+        ("Normandy", "Normandy"),
+        ("Nimbus-Desktop", "Nimbus-Desktop"),
+        (
+            "Nimbus-Mobile",
+            "Nimbus-Mobile",
```

Comment:
We recently added iOS in addition to Fenix(Android) so I'm not sure whether you want to lump both iOS/Android together here or keep them separate?